### PR TITLE
Update ko.xml

### PR DESCRIPTION
**Pull request by Cl0udNine**
- This includes all latest update

### CODE REVIEW

File: src/main/resources/lang/ko.xml [hunk 1]
```diff
@@ -1,157 +1,180 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <locale lang="ko">
     <error>
-        <noMessages>당신에게는 답장을 할 개인 메시지가 없습니다</noMessages>
-        <restartDuringMatch>경기중에는 재시작을 할 수 없습니다. -f를 이용해 강제로 경기를 끝내십시오.</restartDuringMatch>
-        <cycleDuringMatch>경기중에는 사이클 할 수 없습니다. -f로 강제로 경기를 끝내십시오.</cycleDuringMatch>
+        <notEnoughArgsSay>You must have a message to say.</notEnoughArgsSay>
+        <notEnoughArgsBroadcast>You must have a message to broadcast.</notEnoughArgsBroadcast>
```
Comment (TiTo-418) [inline src/main/resources/lang/ko.xml]: English

File: src/main/resources/lang/ko.xml [hunk 2]
```diff
@@ -1,157 +1,180 @@
...
+        <noMessages>당신에게는 답장을 할 개인 메시지가 없습니다.</noMessages>
+        <restartDuringMatch>경기중에는 재시작 할 수 없습니다. -f로 강제로 경기를 끝내십시오</restartDuringMatch>
+        <cycleDuringMatch>경기중에는 이동 할 수 없습니다. -f로 강제로 경기를 끝내십시오</cycleDuringMatch>
         <matchRunning>경기가 이미 진행중입니다.</matchRunning>
-        <matchNoResume>경기는 이미 끝났고 재시작될수 없습니다.</matchNoResume>
+        <matchNoResume>경기는 이미 끝났으며 재시작될수 없습니다.</matchNoResume>
         <matchNoStart>지금은 경기를 시작할수 없습니다.</matchNoStart>
         <matchNoEnd>지금은 경기를 끝낼수 없습니다.</matchNoEnd>
-        <noSetNext>재시작이 대기중인 경우에는 다음맵을 세팅 할 수 없습니다. 강제로 진행하려면 -f 을 사용하십시오.</noSetNext>
+        <notEnoughPlayers>There are not enough players to start the match.</notEnoughPlayers>
```
Comment (TiTo-418) [inline src/main/resources/lang/ko.xml]: English

Comment (Cl0udNine) [inline src/main/resources/lang/ko.xml]: @TiTo-418 Ah, Ill fix it :C

**Comment by Cl0udNine** [comment]
Oops, Ill recreate this PR.

**Comment by twizmwazin** [comment]
don't make a new pr, update the commit, use `git commit --amend --no-edit`

**Pull request closed without merging**